app/models/academic_year.rb

```diff
@@ -1,6 +1,9 @@
 class AcademicYear < ApplicationRecord
   has_many :placement_preferences
 
+  attribute :starts_on, :date
+  attribute :ends_on,   :date
```

Comment:
I have added these attributes in for the meantime to get back the date functionality. Could really use some insight why the migration has disrupted the date fields.